WooCommerce/Classes/ViewRelated/ReusableViews/SpacerTableViewCell.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+/// Used for variable vertical spacing between two table view cells.
+/// The cell currently has default cell background color (`.listForeground`).
+final class SpacerTableViewCell: UITableViewCell {
+    /// The height of the spacer cell.
+    var height: CGFloat = 0 {
+        didSet {
+            heightConstraint.constant = height
+        }
+    }
```

Comment:
Do you think we should consider `configure(height:)` instead so we're consistent (kind of) with the other cells?

Reply:
yup makes sense! updated in: https://github.com/woocommerce/woocommerce-ios/pull/3342/commits/79499744b164335f3b80cad620152502c4d1a96b